14/index.php

```diff
@@ -0,0 +1,174 @@
+<?php
+require_once('includes/template.php');
+require_once('includes/ui/downloads.php');
+require_once('includes/appstore.php');
+require_once('includes/playstore.php');
+
+// Required
+head([
+  'class' => 'keyman11',
+  'title' => 'Keyman 14 is now in Beta!',
+  'css' => ['template.css', 'dev.css', 'app-store-links.css', 'prism.css'],
+  'js' => ['prism.js'],
+  'showMenu' => true,
+  'banner' => [
+    'title' => 'Keyman 14.0 Beta',
+    'image' => 'world.png',
+    'background' => 'water'
+  ]
+]);
+?>
+
+<h2>Announcing the release of Keyman 14.0 Beta</h2>
+<p class="red">30 November 2020</p>
+<p>We are pleased to announce that Keyman 14.0 is now in beta!</p>
+<br>
+
+<h3>What's New?</h3>
+<ul>
+  <li>
+    Simpler and Smoother Keyboard Seach<br>
+    <img alt='Search for Khmer Keyboard' src='keyboard_search_khmer.png'>
+  </li>
+  <li>
+    Localizable UI through <a href="https://translate.keyman.com">translate.keyman.com</a>.
+  </li>
+  <li>
+    Mobile apps download and install keyboard packages from keyman.com
+  </li>
+  <li>
+    Consolidated crash reporting to <a href="https://sentry.keyman.com">sentry.keyman.com</a>
+  </li>
+</ul>
+<br>
+
+<h3>What's Next?</h3>
+<ul>
+  <li>
+    Check the <a href="https://blog.keyman.com/2020/03/keyman-roadmap-march-2020/">Keyman Roadmap</a> for upcoming features.
+  </li>
+</ul>
+<br>
+
+<h3>14.0 Beta Feedback</h3>
+<ul>
+  <li>Please send feedback about Keyman 14 to the
+    <a href="https://community.software.sil.org/t/keyman-14-beta-feedback/4115">Keyman Community site</a> or submit bugs and feature requests to our
+    <a href="https://github.com/keymanapp/keyman/issues/new/choose">Issue Tracker</a></li>
+</ul>
+<br>
+
+<h1 class='red underline'>User Software</h1>
+
+<?php
+downloadSection('Keyman for Windows 14',   'windows',     'keyman-$version.exe', 'beta');
+?>
+
+<h3>What's New in Keyman for Windows 14?</h3>
+
+<ul>
+  <li>Renamed from <strong class="red">Keyman Desktop</strong> to <strong class="red">Keyman for Windows</strong></li>
+  <li>Handles major version upgrades from 10 onward</li>
+  <li>Add GUI for setting configuration options</li>
+</ul>
+
+
+
+<?php
+downloadSection('Keyman 14 for macOS',   'mac',     'keyman-$version.dmg', 'beta');
+?>
```

Comment:
I am wondering if we should add a method of doing this as .md with a special insert marker for these? That would make this page much easier to maintain.